src/assets/database.sql

```diff
@@ -78,6 +78,35 @@ INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `comm
+('Write all valid subdomains to this file.', 'dnsenum --subfile subdomains_obtained.txt example.com', 1, 'input', '--subfile', NULL, NULL, 'GENERAL OPTIONS')
+('The tcp and udp timeout values in seconds, i.e., the interval of time in which dnsenum waits from an answer from the DNS server (default: 10s).', 'dnsenum -t 10 example.com', 1, 'input', '-t', NULL, NULL, 'GENERAL OPTIONS')
+('The number of threads that will perform different queries.', 'dnsenum --threads 5 example.com', 1, 'checkbox', NULL, NULL, 'GENERAL OPTIONS')
+('Be verbose: show all the progress and all the error messages.', NULL, 1, 'checkbox', '-v', NULL, NULL, 'GENERAL OPTIONS')
+--Google Scapping options
+('The number of google search pages to process when scraping names, the default is 5 pages, the -s switch must be specified.', 'dnsenum -p 15 example.com', 1, 'input', '-P', NULL, NULL, 'GOOGLE SCRAPPING OPTIONS')
+('The maximum number of subdomains that will be scraped from Google (default 15)', 'dnsenum -s 40 example.com', 1, 'input', '-s', NULL, NULL, 'GOOGLE SCRAPPING OPTIONS: SUBDOMAINS')
+--Brute force options
+('Read subdomains from this file to perform brute force. (Takes priority over default dns.txt)', NULL, 'dnsenum -f subdomains.txt example.com', 1, 'input', '-f', NULL, NULL, 'SUBDOMAIN BRUTE FORCE')
+('Update the subdomains file specified with the -f switch with valid subdomains. Each parameter for this commmand specifies a source for the updates. Parameters: a (all): Update using all results; g; Update using only google scraping results; r: Update using only reverse lookup results; z: Update using only zonetransfer results.', 'dnsenum -f subdmains.txt -u a example.com', 1, 'input', '-u', NULL, NULL, 'BRUTE FORCE OPTIONS: UPDATE')
+('Recursion on subdomains, brute force all discovered subdomains that have an NS record.', NULL, NULL, 1, 'checkbox', '-r', NULL, NULL, 'BRUTE FORCE OPTIONS: RECURSION')
+--Whois Netrange Options
+('Perform the whois queries on c class network ranges. **Warning**: this can generate very large netranges and it will take lot of time to perform reverse lookups.', 'dnsenum -w example.com', 1, 'checkbox', NULL, NULL, 'WHOIS NETRANGE OPTIONS')
+('The maximum value of seconds to wait between whois queries, the value is defined randomly, default: 3s.', 'dnsenum -w -d 5 example.com', 1, 'input', '-d', NULL, NULL, 'WHOIS NETRANGE OPTIONS: DELAY')
+--Reverse lookup options
+('Exclude PTR records that match the regexp expression from reverse lookup results, useful on invalid hostnames. In this sense, when a reverse lookup is made, subdomains with the defined words are not considered', 'dnsenum -e '(test|local|invalid)' examples.', 1, 'input', '-e', NULL, NULL, 'REVERSE LOOKUP OPTIONS')
+--Output options:
+('Output in XML format. Can be imported in MagicTree (www.gremwell.com)', 'dnsenum -o output.xml example.com', 1, 'input', '-o', NULL, NULL, 'OUTPUT OPTIONS')
```

Comment:
@mayara-santos01 I see that you are using a long "name" and NULL "description" for some cases. I suspect this will break a little bit the visualization. Usually the name is a very short description of the parameter (two or three words at most, they will be displayed when you click +Options), while description is actually the long text (it will be displayed on the Help of the command).

Another thing is: the last line above seems to be missing one field (it has 8 fields, while it is expected to have 9)